plugins/modules/zos_backup_restore.py

```diff
@@ -186,15 +186,15 @@
   hlq:
     description:
       - Specifies the new HLQ to use for the data sets being restored.
-      - Defaults to running user's username.
+      - If no value is provided, the data sets will be restored with their original HLQs.
     type: str
     required: false
   tmp_hlq:
     description:
       - Override the default high level qualifier (HLQ) for temporary and backup
         data sets.
-      - The default HLQ is the Ansible user that executes the module and if
-        that is not available, then the value of C(TMPHLQ) is used.
+      - The default is the original HLQ of the data sets.
+        If that is not available, then the value of C(TMPHLQ) is used.
```

Comment:
I am a bit confused by this and its not your doing, I think a long time ago when the module did not support a user entry for `hlq`, we would honor the ZOAU environment var `TMPHLQ` while we sort of do but not in this case I don't believe. 

So this:
```
 If that is not available, then the value of C(TMPHLQ) is used.
```
sort of contradicts this:
```
The default is the original HLQ of the data sets.
```

I would imagine if 'If that is not available' then the action is no HLQ is used meaning it falls back to `The default is the original HLQ of the data sets.`

Reply:
i believe the desired default behavior from the zoau side of the house is somewhat nebulous and a jira has been (or will be). Figured in the mean time it'd be fine to keep the message close to what the original form was and tweak what has changed, ie default is original not user